src/given/definitions.js

```diff
@@ -125,6 +126,11 @@ const stepDefinitions = [
 		regex: /^a (alertbox|confirmbox|prompt) is( not)* opened$/,
 		fn: checkModal,
 		title: "Check if a modal is opened"
+	},
+	{
+		regex: /^I debug$/,
+		fn: debugGiven,
+		title: "Add a breakpoint to a feature file"
```

Comment:
This title can be more useful: "Add a breakpoint to a feature file" doesn't really describe what it's doing. The language used on http://webdriver.io/api/utility/debug.html is more useful. Also: title renders markdown in the readme, so you can put a link to this wdio docs page.